content/docs/setup/kubernetes/install/multicluster/gateways/index.md

```diff
@@ -43,6 +43,15 @@ on **each** Kubernetes cluster.
   sample root CA certificate available in the Istio installation
   under the `samples/certs` directory.
 
+* This mode of multicluster works best with L4 network load balancers (NLB).
```

Comment:
This is better, but I was thinking that this could be combined with the earlier bullet about accessing the ingressgateway, since they are related. Would a single bullet like this work?
```
* The IP address of the `istio-ingressgateway` service in each cluster must be accessible
  from every other cluster, ideally using L4 network load balancers (NLB).
  Not all cloud providers support NLBs and some require special annotations to use them,
  so please consult your cloud provider’s documentation for enabling NLBs for
  service object type load balancers. When deploying on platforms without
  NLB support, it may be necessary to modify the healthchecks for the load
  balancer to register the ingress gateway.
```

Reply:
good idea - will make it so.

Cheers
-steve